### DISCUSSION TITLE
How to make an admin user organization owner?

### DISCUSSION THREAD

**Discussion by kristiannikolaev** [dani-garcia/vaultwarden#3316] [General]
Hi, I have a docker-compose self-hosted vaultwarden. The organization we have there was created in Bitwarden_rs (already deprecated). That organization was created by a user (the owner) and this user left the company, leaving the organization as the only owner.

Is it possible to change the owner? Maybe connecting to the DB? I really need to change the ownership of the organization

Thank you

**Comment by BlackDex** [comment]
If you have the latest Vaultwarden image right now, you can login into the admin interface and promote a user.

**Reply by kristiannikolaev** [reply]
Oh! Thank you, that solved it!!

**Reply by mathmul** [reply]
[RESOLVED]

Are we talking about "Admin Console" in GUI? If I go to "Members", and "Edit member" (three dots), change to "Owner", and "Save", it says "An error has occurred. Only Owners can grant and remove Admin or Owner privileges "

I'm in the same position as the OP. I have also tried copying `password_hash`, `salt`, `security_stamp`, `totp_secret`, `enabled`, `login_verify_count`, `client_kdf_type`, and `client_kdf_iter` (in that order and testing login with owner account after each) from my (admin) account with which I can login, to the owner account, and it still didn't work.

Is there a different solution?

EDIT:
Oh I also found VaultWarden Admin at bitwarden.domain.com/admin and I can see users in the Users page but only options on the right of each admin are
- Remove all 2FA
- Deauthorize sessions
- Delete User
- Disable User

EDIT2:
I had to click on the organization (a button), and could change there. Leaving all this here for future reference for others.

**Reply by BlackDex** [reply]
You need to do that via the Vaultwarden Admin Interface `/admin`.
